__docs__/wonder-blocks-form/choice.stories.tsx

```diff
@@ -83,7 +104,6 @@ export const Default: StoryComponentType = {
     render: (args) => <ChoiceWrapper {...args} />,
     args: {
         label: "Pineapple (Control)",
-        value: "pineapple",
```

Comment:
Do we need the `value` arg set so the `value` is set on the Pineapple choice on line 57?

https://github.com/Khan/wonder-blocks/blob/b42a7b6ba25e40530c1b7c9854f1bcb75e07ec74/__docs__/wonder-blocks-form/choice.stories.tsx#L57

Reply:
Yes, good catch! Those repeated controls were broken before (one label for two inputs)...I had to go through each prop piece by piece to fix it and missed putting one back!

Reply:
I added this value back in!